.github/workflows/code-review-gpt.yml

```diff
@@ -0,0 +1,26 @@
+name: Code Review GPT (see logs for details)
+
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
Please add environment preview ;) the openai api key is only available for the preview env. 

Check the pytest yaml